.github/workflows/ci.yml

```diff
@@ -64,8 +64,8 @@ jobs:
       run: |
         echo "GIT_USER=Renku Bot" >> $GITHUB_ENV
         echo "GIT_EMAIL=renku@datascience.ch" >> $GITHUB_ENV
-    - name: Push chart and images
-      uses: SwissDataScienceCenter/renku/actions/publish-chart@0.7.11
+    - name: Push tagged chart and images
+      uses: SwissDataScienceCenter/renku/actions/publish-chart@master
```

Comment:
Hmm, I see why you have to do this, and it makes me realize that we should move this action out of the main Renku repo. We don't want to do a Renku release just for the action, but we don't want to use an untagged version here either, as this has broken things before. I'd propose a separate renku-actions repo. @rokroskar what do you think? Or do you have another idea as you have set many of those things up in the first place?

Reply:
I was just reluctant at the time to make yet another repo - but yes, I agree it would make sense if those lived in a separate place.

Reply:
Ok, I've created this repo and tagged a first release that you can refer to in this PR: https://github.com/SwissDataScienceCenter/renku-actions/releases/tag/v0.1.0

Reply:
That is great! Thanks @ableuler!